src/platform/posix.c

```diff
@@ -72,8 +72,13 @@ void platform_get_date_utc(rct2_date *out_date)
 	time(&rawtime);
 	timeinfo = gmtime(&rawtime);
 	out_date->day = timeinfo->tm_mday;
-	out_date->month = timeinfo->tm_mon;
-	out_date->year = timeinfo->tm_year;
+	if (OPENRCT2_PLATFORM == "Windows") {
```

Comment:
It fixed it because the the version of that function in windows.c doesn't require adding 1900.